exampleCourse/questions/template/multiple-choice/all-of-the-above/question.html

```diff
@@ -0,0 +1,11 @@
+<pl-question-panel>
+  <p>Which of the following are considered fundamental programming concepts?</p>
+</pl-question-panel>
+
+<pl-multiple-choice answers-name="cs-concepts" all-of-the-above="correct" number-answers="4">
+  <pl-answer correct="true">Variables</pl-answer>
+  <pl-answer correct="true">Loops</pl-answer>
+  <pl-answer correct="true">Conditionals</pl-answer>
+  <pl-answer correct="false">Graphics</pl-answer>
```

Comment:
Unused wrong answer choices added because of this issue: https://github.com/PrairieLearn/PrairieLearn/issues/12222

Can remove if this gets resolved.